### PR TITLE
Ability to get REBALANCE_LND_FILEPATH and LND_DIR from env.

### PR DESCRIPTION
**Pull request by ibz**
I'm doing this so I can include `perfectly-balanced` in [`lightning-shell`](https://github.com/ibz/lightning-shell).

Thank you!

### CODE REVIEW

**Comment by cuaritas** [comment]
Hello @ibz Thank u for ur contribution 👍

File: perfectlybalanced.sh [hunk 1]
```diff
@@ -7,19 +7,19 @@
...
+VERSION="0.0.9"
 
 FILENAME=$0
 
 MAX_FEE=50 # Sats
 
 TOLERANCE=0.95 # 95%
 
-LND_DIR=$HOME/.lnd/
+LND_DIR="${LND_DIR:-$HOME/.lnd/}"
```
Comment (cuaritas) [inline perfectlybalanced.sh:18]: LGTM

**Review by cuaritas — APPROVED** [review]
LGTM

**Comment by ibz** [comment]
Thank you!

Would you mind creating a tag also?

**Pull request merged**